integration/unplugin/src/index.ts

```diff
@@ -252,6 +256,20 @@ const civetUnplugin = createUnplugin((options: PluginOptions = {}) => {
 
         return null;
       },
+      async transformIndexHtml(html) {
```

Comment:
Should we do this only in dev mode? Or is it fine to have in both?

Reply:
Should be fine in both